openedx/core/djangoapps/user_api/accounts/tests/test_retirement_views.py

```diff
@@ -1482,57 +1523,29 @@ def test_can_retire_users_datasharingconsent(self):
         AccountRetirementView.retire_users_data_sharing_consent(self.test_user.username, self.retired_username)
         self._data_sharing_consent_assertions()
 
-    def _data_sharing_consent_assertions(self):
-        """
-        Helper method for asserting that ``DataSharingConsent`` objects are retired.
-        """
-        self.consent.refresh_from_db()
-        assert self.retired_username == self.consent.username
-        test_users_data_sharing_consent = DataSharingConsent.objects.filter(
-            username=self.original_username
-        )
-        assert not test_users_data_sharing_consent.exists()
-
     def test_can_retire_users_sap_success_factors_audits(self):
```

Comment:
Moved these verification functions to the top of the file. Generally, I get confused when helper and assertion functions are mixed in with all the tests. 

I try to keep things like this:
1. setUp/tearDown functions
2. helper and assert/verification functions
3. test cases

If y'all think I'm being too particular, let me know.